instruqt/distributing-with-replicated/01-preparing-to-use-the-sdk/assignment.md

```diff
@@ -64,7 +63,8 @@ platform.
 export REPLICATED_API_TOKEN="[[ Instruqt-Var key="REPLICATED_API_TOKEN" hostname="shell" ]]"
 ```
 
-And lastly make sure we are working with the Harbor app
+And lastly make sure we are working with the Harbor app we
```

Comment:
in here you use the word `we` twice to refer to two different people. The first time is "we" the users of the lab, and the second time is "we" the creators/hosts of the lab. Could be confusing. Maybe 

```suggestion
And lastly make sure we are working with the Harbor app that was
```